website/static/js/projectSettings.js

```diff
@@ -8,77 +8,114 @@ var $osf = require('js/osfHelpers');
 var oop = require('js/oop');
 var ChangeMessageMixin = require('js/changeMessage');
 
-var NodeCategorySettings = oop.extend(
+var ProjectSettings = oop.extend(
     ChangeMessageMixin,
     {
-        constructor: function(category, categories, updateUrl, disabled) {
+        constructor: function(params) {
             this.super.constructor.call(this);
-
             var self = this;
 
-            self.disabled = disabled || false;
-            self.UPDATE_SUCCESS_MESSAGE = 'Category updated successfully';
-            self.UPDATE_ERROR_MESSAGE = 'Error updating category, please try again. If the problem persists, email ' +
+            self.decodedTitle = params.currentTitle;
+            self.decodedDescription = $osf.htmlDecode(params.currentDescription);
```

Comment:
This variable can probably be removed based on our conversation about changes to the settings.mako template.